sktime/utils/_testing/estimator_checks.py

```diff
@@ -134,7 +135,7 @@ def _make_fit_args(estimator, **kwargs):
     elif isinstance(estimator, BaseSeriesAnnotator):
         X = make_annotation_problem(**kwargs)
         return (X,)
-    elif isinstance(estimator, BaseClassifier):
+    elif isinstance(estimator, (BaseClassifier, BaseEarlyClassifier)):
```

Comment:
I believe this will never be called, so you don´t need to add it here.
You can test it by raising a `RuntimeError` if you arrive here with a `BaseEarlyClassifier`.

`_make_fit_args` is on its way to removal, it is only coupled with exotic scitypes anymore and transformers (until someone reviews my refactor PR 😄 )

Reply:
Removed it. We will figure out if it was required if the tests fail, I guess!